src/app/components/AddEscrowForm/index.tsx

```diff
@@ -61,30 +64,47 @@ export const AddEscrowForm = memo((props: Props) => {
   }, [dispatch])
 
   return (
-    <Form onSubmit={submit}>
-      <Box direction="row" gap="small" pad={{ top: 'small' }}>
-        <Box background="background-front">
-          <TextInput
-            data-testid="amount"
-            id="amount-id"
-            name="amount"
-            placeholder={t('common.amount')}
-            type="number"
-            step="any"
-            min={minStaking}
-            value={amount}
-            onChange={event => setAmount(event.target.value)}
-            required
+    <>
+      {showNotice && (
+        <p>
+          {t(
+            'account.addEscrow.confirmDelegatingToTop.description',
+            'This validator is ranked in the top 20 by stake. Please consider delegating to a smaller validator to increase network security and decentralization.',
```

Comment:
I guess more of a copy question for product, but my first instinct is the softer approach of prodding the user through giving them context.

Reply:
@peterjgilbert

Reply:
i think it's fine :D